src/main/java/io/kestra/plugin/liquibase/Diff.java

```diff
@@ -0,0 +1,141 @@
+package io.kestra.plugin.liquibase;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.runners.TargetOS;
+import io.kestra.core.models.tasks.runners.TaskRunner;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.AbstractExecScript;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.runner.docker.Docker;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Run Liquibase diff to compare DB schemas."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Compare two Postgres databases and generate diff file.",
+            full = true,
+            code = """
+                id: liquibase_diff
+                namespace: company.team
+
+                tasks:
+                  - id: db_diff
+                    type: io.kestra.plugin.liquibase.Diff
+                    url: jdbc:postgresql://pg1:5432/demo1
+                    username: user1
+                    password: pass1
+                    referenceUrl: jdbc:postgresql://pg2:5432/demo2
+                    referenceUsername: user2
+                    referencePassword: pass2
+                    changelogFile: diff.xml
```

Comment:
We should be able to handle [changelogs](https://docs.liquibase.com/pro/user-guide/what-is-a-changelog) as JSON, SQL or YAML too because those format are way more used than XML in production.